.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: python
 python:
-- '2.7'
 - '3.5'
 - '3.6'
+- '3.7'
```

Comment:
Python 3.8 is "official" already.
Should we just add it here?

Reply:
That would make sense in my opinion. Well spotted.

Reply:
There are multiple issues with dependencies on Python 3.8 (see [travis](https://travis-ci.org/github/blue-yonder/devpi-builder/jobs/672443191)).
I would propose moving forward with this PR and looking into the issues connected to Python 3.8 in another PR.

Reply:
So much for my statement regarding surprises :D ...
Well, thanks for trying :D...